agent/util-scripts/pbench-sysinfo-dump

```diff
@@ -123,7 +123,7 @@ function collect_sos {
 	for mod in ${_modules}; do
 		printf -- " -o %s" "${mod}" >> ${_cmd}
 	done
-	printf -- "--batch%s --tmp-dir=\"%s\" --name \"%s\"" "${_quiet}" "${dir}" "${_name}" >> ${_cmd}
+	printf -- " --batch%s --tmp-dir=\"%s\" --name \"%s\"\n" "${_quiet}" "${dir}" "${_name}" >> ${_cmd}
```

Comment:
I get the space but what's with the `\n`? That looks wrong to me.

Reply:
Never mind...

Reply:
Oh, yeah; I puzzled over that for a moment, too; but it's the end of the command, so terminating the line makes sense.